mw-webapp/src/utils/unicodeSymbols.ts

```diff
@@ -1,4 +1,15 @@
 /**
- * Zero-width space by unicode
+ * Unicode symbols
  */
-export const SPACE = "\u200B";
\ No newline at end of file
+export enum unicodeSymbols {
+
+  /**
+   * Zero-width space by unicode
+   */
+  space = "\u200B",
```

Comment:
if it is not regular space - why it called space?

Reply:
Renamed